index.bs

```diff
@@ -638,6 +645,11 @@ agent MUST run the following steps:
    {{TypeError}}.
 1. Let |requireUnreliable| be {{WebTransport/constructor(url, options)/options}}'s
    {{WebTransportOptions/requireUnreliable}}.
+1. Let |congestionControl| be {{WebTransport/constructor(url, options)/options}}'s
+   {{WebTransportOptions/congestionControl}}.
+1. If the user agent only implements the default congestion control algorithm in
+   [[!RFC9002]] [section 7](https://datatracker.ietf.org/doc/html/rfc9002#section-7),
```

Comment:
that's fairly narrowly targeted, so maybe "... or an algorithm that optimizes for throughput ..."